templates/projects/message-listener-with-api-client-and-server/src/main/kotlin/uk/gov/justice/digital/hmpps/controller/ApiController.kt

```diff
@@ -0,0 +1,17 @@
+package uk.gov.justice.digital.hmpps.controller
+
+import org.springframework.security.access.prepost.PreAuthorize
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+class ApiController {
+    @PreAuthorize("hasRole('ROLE_EXAMPLE')")
+    @GetMapping(value = ["/example/{inputId}"])
+    fun handle(
+        @PathVariable("inputId") inputId: String
+    ) {
+        // TODO Not yet implemented
```

Comment:
Thanks.  It was just to avoid an ugly try-catch in the example integration test.  Agree it's not ideal that it's inconsistent though - might be worth switching the other TODO() calls to the comment version at some point in future, then we can remove the other try-catch blocks.

I'll merge this as is for now though.